appcues/src/main/java/com/appcues/trait/appcues/TooltipPathExt.kt

```diff
@@ -166,6 +160,7 @@ internal fun getTooltipSettings(
     distance: Dp,
     pointerBaseDp: Dp,
     pointerLengthDp: Dp,
+    pointerCornerRadius: Dp = 0.dp,
```

Comment:
do we need a default value here any longer? I think it's always passed in now.

Reply:
nice catch, not being used anymore